calculation/sha_calc/sha_calc/directivity/bea20/bea20.py

```diff
@@ -31,37 +31,36 @@ def bea20(
         Moment magnitude, 5<=M<=8
     U: np.ndarray
         The GC2 coordinates in km. Equivalent to ry.
-        Must be nX1 number where n is the number of locations at which the model provides a prediction.
+        A 1d array of length n where n is the number of locations at which the model provides a prediction.
     T: np.ndarray
         The GC2 coordinates in km. Equivalent to rx.
-        Must be nX1 number where n is the number of locations at which the model provides a prediction.
-    Smax: List
-        List of 2x1 number where first is the lowest max value and the second is the largest max value for s.
+        A 1d array of length n where n is the number of locations at which the model provides a prediction.
+    Smax: Tuple[float, float]
+        Tuple of 2x1 float where the first value is the lowest max value and the second is the largest max value for s.
         Computed from GC2 using nominal strike of the fault.
     D: float
-        The effective rupture travel width, measured from the hypocenter to the shallowest depth of the rupture plane.
+        The effective rupture travel width, measured from the hypocentre to the shallowest depth of the rupture plane.
     Tbot: float
         The length of the bottom of the rupture plane (projected to the surface) in km.
     Dbot: float
         The vertical depth of the bottom of the rupture plane from the ground surface, including Ztor, in km.
     Rake: float
         The Rake of the fault
     Dip: float
-        The Dip of the fault at the hypocenter
+        The Dip of the fault at the hypocentre
     Period: float
         Used for specifying the period for a pSA IM
-    type: float, optional
-        0 for type base don strike
+    rupture_type: float, optional
```

Comment:
Is it overkill to use an enum here instead of magic numbers?

Reply:
Ain't no kill like overkill

Reply:
Maybe in other cases, however in this I think it makes sense to keep the inputs to model in-line with the original matlab code